tests/extrusion/test_variable_layers_numbering.py

```diff
@@ -119,15 +119,15 @@ def test_numbering_one_d_P3():
 
     assert numpy.equal(V.cell_node_map().values,
                        [[21, 24, 22, 23, 14, 17, 15, 16,
-                         0, 6, 2, 3, 1, 7, 4, 5],
+                         1, 7, 4, 5, 0, 6, 2, 3],
                         [48, 51, 49, 50, 27, 30, 28, 29,
-                         34, 40, 36, 37, 35, 41, 38, 39]]).all()
+                         35, 41, 38, 39, 34, 40, 36, 37]]).all()
 
     assert numpy.equal(V.exterior_facet_node_map().values,
                        [[21, 24, 22, 23, 14, 17, 15, 16,
-                         0, 6, 2, 3, 1, 7, 4, 5],
+                         1, 7, 4, 5, 0, 6, 2, 3],
                         [48, 51, 49, 50, 27, 30, 28, 29,
-                         34, 40, 36, 37, 35, 41, 38, 39]]).all()
+                         35, 41, 38, 39, 34, 40, 36, 37]]).all()
```

Comment:
I gues these changes are consequences of the renumbering on the interior of the interval cells?

Reply:
Yes, exactly.